android/sdl_android/src/main/java/com/smartdevicelink/encoder/VirtualDisplayEncoder.java

```diff
@@ -107,10 +107,16 @@ public VideoStreamingParameters getStreamingParams(){
     }
 
     @SuppressWarnings("unused")
+    @Deprecated
     public void setStreamingParams(int displayDensity, ImageResolution resolution, int frameRate, int bitrate, int interval, VideoStreamingFormat format) {
         this.streamingParams = new VideoStreamingParameters(displayDensity, frameRate, bitrate, interval, resolution, format);
     }
 
+    @SuppressWarnings("unused")
+    public void setStreamingParams(int displayDensity, ImageResolution resolution, int frameRate, int bitrate, int interval, double scale, VideoStreamingFormat format) {
```

Comment:
For any new public method added, can we please have some javadocs with them? just the params with a brief description should be adequate.

Reply:
Thanks a lot for the review,
now  I realize I should not modify this method nor VideoStreamingParameters. The scale value is coming from the head.
By adding the scale parameter in setStreamingParams, I was allowing the user to modify the scale, which is wrong.

Reply:
alright, sounds good. please let me know when you are ready for re-review. We have our machine set up so we shouldn't have any weird VM issues this time. 👍

Reply:
thanks, Ready for re review